tool/lsm.go

```diff
@@ -288,6 +293,9 @@ func (l *lsmT) buildEdits(edits []*manifest.VersionEdit) error {
 
 		for j := range ve.NewFiles {
 			nf := &ve.NewFiles[j]
+			if b, ok := backings[nf.BackingFileNum]; ok {
```

Comment:
nit: probably helpful to throw this behind an `if nf.Meta.Virtual` just to avoid some zero value `nf.BackingFileNum` cases. but also alright as-is.

Reply:
done.